### PR TITLE
Set num workers to 0 for rnnt single gpu run

### PR DESCRIPTION
**Pull request by pnunna93**
This PR has changes to set num_workers to 0 for rnnt single gpu runs. This change helped improve performance on ROCm.

### CODE REVIEW

**Review by dllehr-amd — APPROVED** [review]
Looking good on my end!

File: benchmarks/rnnt/ootb/train/scripts/train.sh [hunk 1]
```diff
@@ -87,6 +88,7 @@ ARGS+=" --grad_accumulation_steps=$GRAD_ACCUMULATION_STEPS "
 ARGS+=" --device=$DEVICE"
 ARGS+=" --beta1=$BETA1"
 ARGS+=" --beta2=$BETA2"
+ARGS+=" --num-workers=${NUM_WORKERS}"
```
[the remark this answers is not in the document]
Comment (pnunna93) [reply benchmarks/rnnt/ootb/train/scripts/train.sh]: Removed here: https://github.com/facebookresearch/FAMBench/pull/117/commits/7f68c9cb224cdcbf87a2fd5cf46649f5a3960d37

**Review by amathews-amd — APPROVED** [review]
LGTM

**Comment by amathews-amd** [comment]
@pnunna93 , please resolve conflicts

**Comment by pnunna93** [comment]
> @pnunna93 , please resolve conflicts

Fixed it

**Review by amathews-amd — APPROVED** [review]
LGTM

**Pull request merged**